csrc/scheduler/resize.cpp

```diff
@@ -472,9 +471,8 @@ void ResizeScheduler::schedule(Fusion* fusion, const HeuristicParams* params) {
   }
 
   if (vec_factor > 1) {
-    auto vec_ref_tv = largest_input != nullptr ? largest_input : ref_tv;
     const auto tvs_to_vectorize =
-        scheduler_utils::getInputsOutputsWithInnerDim(vec_ref_tv, true, true);
+        scheduler_utils::getInputsOutputsWithInnerDim(ref_tv, true, true);
```

Comment:
Can you remind me why we were using `largest_input` as vectorization reference in the first place?

Reply:
Forgot to mention this, but this was an actually a bug. It should have been changed in #3955.